src/cloudant/_common_util.py

```diff
@@ -234,6 +234,24 @@ def codify(code_or_str):
         return _Code(code_or_str)
     return code_or_str
 
+def _get_docs(url, **kwargs):
+    """
+    Provides a helper to get a response with documents for GET/POST requests.
```

Comment:
Reading this comment I wouldn't know when it was ok to use this utility (even internally).